model/user-agent/registry.yaml

```
@@ -35,3 +35,20 @@ groups:
           using a user-agent for non-browser products, such as microservices with multiple names/versions inside the
           `user_agent.original`, the most significant version SHOULD be selected. In such a scenario it should align
           with `user_agent.name`
+      - id: user_agent.synthetic
+        stability: experimental
+        brief: >
+          A flag indicating that the user agent represents a synthetic source and did not originate from genuine client traffic.
```

Most of the traffic probably happens between services. Can we clarify what the synthetic means?

My interpretation is:

```suggestion
         Specifies the category of synthetic traffic, such as monitoring, crawler, bot, or another automation.
```